File: 119-pascals-triangle-ii.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        row = [1]
        for _ in xrange(1, rowIndex+1):
            for idx in xrange(len(row)-1, 0, -1):
                row[idx] = row[idx] + row[idx-1]
            row.append(1)
        return row

File: 119-pascals-triangle-ii.go
func getRow(rowIndex int) []int {
    row := []int{1}
    for idx := 1; idx <= rowIndex; idx += 1{
        for jdx := len(row)-1; jdx > 0; jdx -= 1{
            row[jdx] = row[jdx] + row[jdx-1]
        }
        row = append(row, 1)
    }
    return row
}
